### PR TITLE
Stuff.

### PR DESCRIPTION
**Pull request by PizzaCrust**
- Fix the POM
- Fancy up `ServerStartEvent`.
  - Replaced server.properties stuff to `ServerInfo`.
  - Moved launch arguments to `Server`.
- Made `BlockBreakEvent` cancellable. 
  - Added `CancellableBase`, since that is most likely the intended implementation.
- Added 'author' at the `@Plugin` annotation.
- Addition in the `Location` class.

### CODE REVIEW

File: src/main/java/org/fountainmc/api/Server.java [hunk 1]
```diff
@@ -4,11 +4,9 @@
 
 public interface Server {
 
-    public String getName();
-    
-    public String getVersion();
-    
-    public String getMotd();
-    
+    public ServerInfo getServerInfo();
```
Comment (Techcable) [inline src/main/java/org/fountainmc/api/Server.java:7]: Can we have server extend ServerInfo?

Comment (ghost) [inline src/main/java/org/fountainmc/api/Server.java:7]: Ok.

**Comment by phase** [comment]
That pom spelling error explains a lot...

I also need to add a CONTRIBUTING file that explains _how to write a good commit message_.

**Comment by phase** [comment]
Merge with https://github.com/FountainMC/FountainAPI/commit/ca7d94ad146c0e5ab1ea6c965dffe81165bcd227, https://github.com/FountainMC/FountainAPI/commit/29737f4e5bd63e935fa889f32cf67f247efe6e48, and https://github.com/FountainMC/FountainAPI/commit/062434c864a56c2773e829667d098fde3b663ddd.

**Pull request closed without merging**